db/migrate/20200223191535_create_rounds.rb

```diff
@@ -0,0 +1,12 @@
+class CreateRounds < ActiveRecord::Migration[5.2]
```

Comment:
not sure why this is being passed in... Could you explain this to me before we merge?